framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1285,60 +1301,66 @@ wifi_manager_result_e _handler_on_scanning_state(_wifimgr_msg_s *msg)
 void _handle_user_cb(_wifimgr_usr_cb_type_e evt, void *arg)
 {
 	WM_LOG_START;
-	wifi_manager_cb_s *cbk = &g_manager_info.cb;
-	switch (evt) {
-	case CB_STA_CONNECTED:
-		nvdbg("[WM] call sta connect success event\n");
-#ifdef CONFIG_ENABLE_IOTIVITY
-		__tizenrt_manual_linkset("gen");
-#endif
-		cbk->sta_connected(WIFI_MANAGER_SUCCESS);
-		break;
-	case CB_STA_CONNECT_FAILED:
-		nvdbg("[WM] call sta connect fail event\n");
-		WIFIADD_ERR_RECORD(ERR_WIFIMGR_CONNECT_FAIL);
-		cbk->sta_connected(WIFI_MANAGER_FAIL);
-		break;
-	case CB_STA_DISCONNECTED:
-		nvdbg("[WM] call sta disconnect event\n");
+	int i = 0;
+	for (; i < WIFIMGR_NUM_CALLBACKS; i++) {
+		wifi_manager_cb_s *cbk = g_manager_info.cb[i];
+		if (!cbk) {
+			continue;
+		}
+		switch (evt) {
+		case CB_STA_CONNECTED:
+			nvdbg("[WM] call sta connect success event\n");
 #ifdef CONFIG_ENABLE_IOTIVITY
-		__tizenrt_manual_linkset("del");
+			__tizenrt_manual_linkset("gen");
```

Comment:
Please correct indentation.

Reply:
Current indentation is correct, considering below lines. Thanks.